src/frontend/components/administration/apikey/CreateDialog.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <v-dialog :value="visible" max-width="1000px" @click:outside="closeDialog()">
```

Comment:
like this ? 
```js
<v-dialog :value="visible" max-width="1000px" @input="updateAttributes" @click:outside="closeDialog()">
```

Reply:
Much like
```html
<v-dialog :value="visible" max-width="1000px" @input="closeDialog">
```
as it will allow you to use `v-model` on your `CreateDialog` (see https://v2.vuejs.org/v2/guide/components.html#Using-v-model-on-Components)